de-lib/src/main/java/org/iplantc/de/tools/client/presenter/ManageToolsViewPresenter.java

```diff
@@ -89,7 +114,7 @@ public void onSuccess(List<Tool> tools) {
     }
 
     @Override
-    public void addTool(Tool tool) {
+    public void addTool(Tool tool, final Command dialogCallbackCommand) {
```

Comment:
I hadn't seen this in our code before... I'll have to keep this pattern in mind 👍